lib/plugins/Label_44_POS.test.ts

```diff
@@ -39,11 +39,11 @@ test('decodes Label 44 Preamble POS02 variant 1', () => {
   expect(decodeResult.formatted.items[0].code).toBe('POS');
   expect(decodeResult.formatted.items[0].label).toBe('Aircraft Position');
   expect(decodeResult.formatted.items[0].value).toBe('38.285 N, 77.845 W');
-  expect(decodeResult.formatted.items[1].type).toBe('origin');
+  expect(decodeResult.formatted.items[1].type).toBe('icao');
   expect(decodeResult.formatted.items[1].code).toBe('ORG');
   expect(decodeResult.formatted.items[1].label).toBe('Origin');
   expect(decodeResult.formatted.items[1].value).toBe('KJFK');
-  expect(decodeResult.formatted.items[2].type).toBe('destination');
+  expect(decodeResult.formatted.items[2].type).toBe('icao');
```

Comment:
Should we not instead do destination_icao?

Reply:
so i'm trying to make the item as follows
type - kind of thing it is - time of day, time since epoch, icao code, etc
code - unique identifier for what it is
label - display of what it is
value - display of the actual value

so in this case it's
type: icao (4-character airport code)
code: DST
label: Destination Airport
value: KUZA

Open to suggestions. will keep this open to discuss

Reply:
I think you're right here. Maybe we need an additional element called name?

Reply:
mmmm - i see the label as the "human name" and code as the "computer name" and the type as a "units". IDK, it's gotten messy and i tried to commandeer type because 3 variations of "destination" or any other value seemed excessive

Reply:
i wrote this and then there's also the raw value. *sigh*